go/libkb/keyfamily.go

```diff
@@ -793,6 +794,8 @@ func (ckf *ComputedKeyFamily) RevokeKid(kid keybase1.KID, tcl TypedChainLink) (e
 		info.Status = KeyRevoked
 		info.RevokedAt = TclToKeybaseTime(tcl)
 		info.RevokedBy = tcl.GetKID()
+		tmp := tcl.ToSigChainLocation()
+		info.RevokedAtSigChainLocation = &tmp
```

Comment:
Here's what we were missing. You can see these lines in `RevokeSig` immediately above. I'm not sure why we didn't add them here.